DigitalLearningSolutions.Data.Tests/DataServices/FaqsDataServiceTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.DataServices
+{
+    using System;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Models.Support;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using FizzWare.NBuilder;
+    using FluentAssertions;
+    using Microsoft.Data.SqlClient;
+    using NUnit.Framework;
+
+    public class FaqsDataServiceTests
+    {
+        private SqlConnection connection = null!;
+        private FaqsDataService faqsDataService = null!;
+
+        [SetUp]
+        public void Setup()
+        {
+            connection = ServiceTestHelper.GetDatabaseConnection();
+            faqsDataService = new FaqsDataService(connection);
+        }
+
+        [Test]
+        public void GetPublishedFaqByIdForTargetGroup_returns_published_faq_with_correct_target_group()
+        {
+            // Given
+            const int faqId = 112;
+            const int targetGroup = 0;
+            var expectedFaq = Builder<Faq>.CreateNew().With(f => f.FaqId = 112).And(f => f.TargetGroup = 0)

```

Comment:
You could just use a constructor here since you're populating all of the fields, might be a bit tidier.

Reply:
Fair enough. I think sticking to one convention might be better in the long run but in these circumstances it certainly is "cleaner" to use a constructor.

Reply:
Yeah, I'll try and write a doc to make our convention more set-in-stone.